nav2_dwb_controller/dwb_core/include/dwb_core/trajectory_generator.hpp

```diff
@@ -126,9 +126,12 @@ class TrajectoryGenerator
 
   /**
    * @brief Limits the maximum linear speed of the robot.
-   * @param speed_limit expressed in percentage from maximum robot speed.
+   * @param percentage Setting speed limit in percentage if true
+   * or in absolute values in false case.
+   * @param speed_limit expressed in absolute value (in m/s)
+   * or in percentage from maximum robot speed.
    */
-  virtual void setSpeedLimit(const double & speed_limit) = 0;
+  virtual void setSpeedLimit(const bool & percentage, const double & speed_limit) = 0;
```

Comment:
As well all here - lets do speed as the first argument and the bool for percentage second

Reply:
Updated in https://github.com/ros-planning/navigation2/pull/2149/commits/6bd018bee773e8ae042062cde5963646a93cba25